src/js/containers/Buttons.js

```diff
@@ -15,14 +15,14 @@ const mapStateToProps = (state) => {
     var app = {}
     var graphicPresent
     if (activeApp) {
-        subscribedButtons = state.ui[activeApp].subscribedButtons
+        subscribedButtons = state.ui[activeApp].subscribedButtons;
```

Comment:
```suggestion
        subscribedButtons = state.ui[activeApp].subscribedButtons
```

Reply:
@jacobkeeler thank you! It has been changed [here](https://github.com/LuxoftSDL/generic_hmi/pull/26/commits/c1a08c356f3f507001a80145f48d31b33c44b097).